frontend/src/services/MoralisService.ts

```diff
@@ -8,7 +8,7 @@ import NetworkService from "./NetworkService";
 
 const MoralisService = {
     getLeaderboard: async function (tokenId: string, contractAddress: string) {
-        return NetworkService.getResourceWithAuth("/v1/moralis/leaderboard/" + tokenId + "/" + contractAddress);
+        return NetworkService.getResourceWithAuth("/v1/nfg/leaderboard/moralis/" + tokenId + "/" + contractAddress);
```

Comment:
I think this is a typo that will break the leaderboard, `/v1/nfg/leaderboard/moralis/` should be -> `/v1/nft/leaderboard/moralis/`

Reply:
fixed now, thanks!